sdk/trace/batch_span_processor.go

```diff
@@ -216,6 +218,7 @@ func (bsp *batchSpanProcessor) exportSpans(ctx context.Context) error {
 	}
 
 	if l := len(bsp.batch); l > 0 {
+		debug.Log.V(5).Info("exporting spans", "count", len(bsp.batch))
```

Comment:
Do we want to use `logr` API directly like here? The API surface is quite large. How do we ensure that we follow some conventions? Do we want to create some abstraction layer to not use `logr` directly? Do we want to create our own linter to e.g. not allow `V(100)` or is it not a problem?

Reply:
I answered it [above](https://github.com/open-telemetry/opentelemetry-go/pull/2343#discussion_r746071510).  TLDR:
We don't have a convention, and we probably won't until we instrument things.  When we get to that point we can look at how to enforce it.

Reply:
Answered in recent [Update](https://github.com/open-telemetry/opentelemetry-go/pull/2343#issuecomment-971775390).  

TLDR:
Did a survey of the `logr` ecosystem, and there is a simple convention that came out of it.